.github/workflows/run.yml

```diff
@@ -25,12 +25,29 @@ jobs:
         echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
         sudo udevadm control --reload-rules
         sudo udevadm trigger --name-match=kvm
+    - name: Build the project
+      run: dotnet build AndroidSdk.Tool
+    # Set ANDROID_SDK_HOME to a consistent location because the AVD manager respects the XDG_CONFIG_HOME
+    # but the emulator does not and fails.
+    - name: Set the ANDROID_SDK_HOME envvar to '~/.android' if it is not already set
```

Comment:
Set `ANDROID_SDK_HOME` to a consistent location because the AVD manager respects the `XDG_CONFIG_HOME` but the emulator does not and fails.

```
INFO         | Android emulator version 35.3.12.0 (build_id 12990079) (CL:N/A)
INFO         | Graphics backend: gfxstream
ERROR        | Unknown AVD name [test], use -list-avds to see valid list.
ERROR        | HOME is defined but there is no file test.ini in $HOME/.android/avd
ERROR        | (Note: Directories are searched in the order $ANDROID_AVD_HOME, $ANDROID_SDK_HOME/avd and $HOME/.android/avd)
```

Reply:
No idea why the override ANDROID_SDK_HOME  or even ANDROID_AVD_HOME is not working. I am just resigning myself to unset the value for the create step.